contrib/Docker_Grafana_Blueflood/docker-entrypoint.sh

```diff
@@ -37,6 +37,8 @@ blueflood:
   apikey: $RAX_APIKEY                     
   authentication_module: rax_auth
   authentication_class: BluefloodAuth
+  enable_submetrics: true
+  submetric_aliases: { "_sum": 'sum' }
```

Comment:
Should we enable the other submetrics too? Or are we only interested in solving the API Availability use case?